pwa/src/templates/componentDetail/ComponentsDetailTemplate.module.css

```diff
@@ -122,7 +122,14 @@
   max-width: unset !important;
 }
 
+.tabBadgeContainer {
+  display: flex;
+  flex-direction: row;
+}
+
 .badgeLayout {
+  height: fit-content;
+  --utrecht-badge-counter-padding-inline: 11px;
```

Comment:
I think this `--utrecht-badge-counter-padding-inline` should hook into one of the following:

- A readily available size, e.g. `--web-app-size-sm` or
- A specifically created token for this badge layout padding, something like `--web-app-badge-layout-....`

Reply:
Couple of notes regarding the layout of this implementation (see screenshots, before and after).

- The text and badge counter are not aligned;
- The badge counter is not a perfect circle.

_Before_
![image](https://user-images.githubusercontent.com/66552932/229509769-87350674-14c9-40ed-b4b0-ea573dda533f.png)

_After (note: just a quick example of what it could be, feel free to edit)_
![image](https://user-images.githubusercontent.com/66552932/229510084-dc090e27-6aa8-462c-a141-99b0afaa8c49.png)